internal/cypress/docker/docker.go

```diff
@@ -254,6 +265,17 @@ func (handler *Handler) StartContainer(ctx context.Context, c cypress.Project) (
 	return &container, nil
 }
 
+// copyTestFiles copies the files within the container.
+func copyTestFiles(ctx context.Context, handler *Handler, containerID string, files []string, pDir string) error {
+	for _, file := range files {
+		log.Info().Str("from", file).Str("to", pDir).Msg("File copied")
```

Comment:
Maybe for better formatting we need to add spaces after `from` and `to` ?

Reply:
Formatting is managed by the library, and `Str()` is just here to provide a set of variables.

Here is an example of output:
```
12:28AM INF File copied from=cypress.json to=/home/seluser/
12:28AM INF File copied from=cypress to=/home/seluser/
```

Reply:
Looks good!